converter/dicom-cast/src/Microsoft.Health.DicomCast.Core/Features/Worker/DicomCastWorker.cs

```diff
@@ -100,6 +109,23 @@ public async Task ExecuteAsync(CancellationToken cancellationToken)
         {
             LogUnhandledExceptionDelegate(_logger, ex);
 
+            if ((ex is FhirException) && ((FhirException)ex).StatusCode == HttpStatusCode.Forbidden)
+            {
+                _telemetryClient.GetMetric("Cast-To-Fhir-Forbidden").TrackValue(1);
```

Comment:
Would it make sense to keep these strings - `Cast-To-Fhir-Forbidden` , `Dicom-To-Cast-Forbidden` in a constants file or something? So they can be referenced from elsewhere, like in the unit tests